packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -217,9 +217,12 @@ export class Combobox extends FormAssociatedCombobox {
      * @public
      */
     @observable
-    public position: SelectPosition = SelectPosition.below;
-    protected positionChanged() {
-        this.positionAttribute = this.position;
+    public position?: SelectPosition;
```

Comment:
@simonxabris @radium-v

This PR was originally targeting the select and is now instead targeting the combobox. Related, the following select-specific PR was marked as being closed by this PR which no-longer updates the select: https://github.com/microsoft/fast/issues/5123

Also, should the position behavior be aligned between the combobox and select, ie the default value and type for position on both should be aligned.

Reply:
Yeah I'm not sure what happened here - the commits view does have the commits where this is applied to both Select and Combobox, but the Files Changed view is only showing changes to the Combobox.